scripts/config/cspell/dictionaries/qiskit.txt

```diff
@@ -356,6 +357,7 @@ unphysical
 upskilling
 varepsilon
 varnothing
+varphi
```

Comment:
Probably shouldn't go in the global dictionary

Reply:
We have a lot of `varphi` in the repo so I think we'd better keep it